ovs/extensions/healthcheck/alba/alba_health_check.py

```diff
@@ -528,36 +517,35 @@ def get_disk_safety(logger):
         Send disk safety for each vpool and the amount of namespaces with the lowest disk safety to DB
         """
 
-        logger.info("Checking if objects need to be repaired...")
-
-        points = []
-
+        logger.info('Checking if objects need to be repaired...')
         test_name = 'disk-safety'
+        abm_services = set(service for service in ServiceHelper.get_services() if service.type.name == ServiceType.SERVICE_TYPES.ALBA_MGR)
 
-        abm_services = set(service for service in ServiceHelper.get_services() if
-                           service.type.name == ServiceType.SERVICE_TYPES.ALBA_MGR)
-
+        disk_lost_overview = {}
         for abm_service in abm_services:
-            alba_backend = BackendHelper.get_albabackend_by_guid(abm_service.abm_service.alba_backend_guid)
+            alba_backend = abm_service.abm_service.alba_backend
+            if alba_backend.name in disk_lost_overview:
+                continue
             # Determine if services are from ab instance
             config = Configuration.get_configuration_path('ovs/arakoon/{0}-abm/config'.format(alba_backend.name))
             # Fetch alba info
+            presets = []
             try:
                 try:
                     # @TODO add this to extra_params to include errored asds. Currently there is a bug with it
                     # Ticket: https://github.com/openvstorage/alba/issues/441
-                    # extra_params=["--include-errored-as-dead"]
-                    namespaces = AlbaCLI.run(command="get-disk-safety", config=config)
+                    # extra_params=['--include-errored-as-dead']
+                    namespaces = AlbaCLI.run(command='get-disk-safety', config=config)
                 except Exception as ex:
-                    raise SystemError("Could not execute 'alba get-disk-safety'. Got {0}".format(ex.message))
+                    raise AlbaException(str(ex), 'get-disk-safety')
```

Comment:
A wrapper to execute ALBA commands would be nice. Because you raise 16 AlbaExceptions in this file alone.